libs/template/src/lib/render-strategies/strategies/global.strategy.ts

```diff
@@ -13,14 +13,20 @@ export function getGlobalStrategies(
 }
 
 /**
+ *
+ * @description
+ *
  * Global Strategies
  *
- * - ɵMD - `ɵmarkDirty`
- * - C - `Component`
+ * This strategy leverages Angular's internal `ɵmarkDirty` render method.
+ * This means it acts identical to `ChangeDetectorRef#markForCheck` but works also zone-less.
  *
- * | Name        | ZoneLess | Render Method | ScopedCoalescing | Scheduling | Chunked |
- * |-------------| ---------| --------------| ---------------- | ---------- |-------- |
- * | `global`     | ✔        | ɵMD           | C + Pr          | ❌         | ❌      |
+ * | Name      | Zone Agnostic | Render Method   | Coalescing    | Scheduling |
+ * | --------- | --------------| --------------- | ------------- | ---------- |
+ * | `global`  | ✔             | `ɵmarkDirty`    | ❌            | ❌         |
```

Comment:
```suggestion
 * | `global`  | ✔             | `ɵmarkDirty`    | ✔            | `requestAnimationFrame`  |
```

look at my comment above. It does scheduling. Also it coalesces, but on `ROOT_CONTEXT` level.

Scheduling happens on `animationFrame` when exists. See: https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/util/misc_utils.ts#L39

```ts
export const defaultScheduler =
    (() => (
               typeof requestAnimationFrame !== 'undefined' &&
                   requestAnimationFrame ||  // browser only
               setTimeout                    // everything else
               )
               .bind(global))();
```

Reply:
The table was targetting rx-angular behavior. Anyhow, I think it looks more complete and is easier to grasp if we add the information also for angular internals!

THX! I'll update the docs!

Reply:
yes i know, i thought it would be helpful information for anyone reading this :)

Reply:
Could you please gather the links to the source?

Reply:
Isn't it the same as in the comment above?! Or what do you mean?

Reply:
place of coalescing and other relevant places
I will include detailed information for both rx-angular and angular internals.

Reply:
Done! Please have another look to all changes